src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -479,6 +474,8 @@ bool mpuDetect(gyroDev_t *gyro, const gyroDeviceConfig_t *config)
 
 void mpuGyroInit(gyroDev_t *gyro)
 {
+    gyro->accDataReg = MPU_RA_ACCEL_XOUT_H;
```

Comment:
For the majority of gyros this assignment is used, but for others, such as the ICM426xx driver, this can be set independently in a custom `initFn`, and, if required, a custom `readFn` can be provided for both the gyro and acc.